classes/Task/Upgrade/BackupDb.php

```diff
@@ -271,39 +240,134 @@ public function run(): int
         );
         $this->container->getFileConfigurationStorage()->save($tablesToBackup->dump(), UpgradeFileNames::DB_TABLES_TO_BACKUP_LIST);
 
+        if ($numberOfSyncedTables) {
+            $this->logger->info($this->translator->trans('%s tables have been saved.', [$numberOfSyncedTables]));
+        }
+
         if ($tablesToBackup->getRemainingTotal()) {
-            $this->logger->debug($this->translator->trans('%s tables have been saved.', [$found]));
             $this->next = 'backupDb';
             $this->stepDone = false;
-            $this->logger->info($this->translator->trans('Database backup: %s table(s) left...', [$tablesToBackup->getRemainingTotal()]));
+            if ($numberOfSyncedTables) {
+                $this->logger->info($this->translator->trans('Database backup: %s table(s) left...', [$tablesToBackup->getRemainingTotal() + (int) !empty($row)]));
```

Comment:
why did you add (int) !empty($row)? Was the value incorrect before?

Reply:
I got errors from PHP complaining about adding boolean to an integer, but I can't reproduce it in a sandboxed env. I'll have another look at it and will remove it if possible.